ko/django_forms/README.md

```diff
@@ -224,16 +224,11 @@ from django.shortcuts import redirect
 
 {% filename %}blog/views.py{% endfilename %}
 ```python
-from django.shortcuts import redirect
+return redirect('post_detail', pk=post.pk)
 ```
 
 `post_detail`은 우리가 이동해야 할 뷰의 이름이에요 *post_detail 뷰* 는 `pk`변수가 필요한 거 기억하고 있겠죠? `pk=post.pk`를 사용해서 뷰에게 값을 넘겨줄 거에요. 여기서 `post`는 새로 생성한 블로그 글이에요.
 
-{% filename %}blog/views.py{% endfilename %}
-```python
-return redirect('post_detail', pk=post.pk)
-```
-
 잘했어요. 너무 설명이 길어졌네요. 이제 *view* 전체 코드를 확인할게요.
```

Comment:
[Fix Mistranslation]
Here is Original Text:

> Finally, it would be awesome if we could immediately go to the post_detail page for our newly created blog post, right? To do that we need one more import:
> 
> ```python
> # blog/views.py
> from django.shortcuts import redirect
> ```
> Add it at the very beginning of your file. And now we can say, "go to the post_detail page for the newly created post":
> 
> ```python
> # blog/views.py
> return redirect('post_detail', pk=post.pk)
> ```
> post_detail is the name of the view we want to go to. Remember that this view requires a pk variable? To pass it to the views, we use pk=post.pk, where post is the newly created blog post!
> 
> OK, we've talked a lot, but we probably want to see what the whole view looks like now, right?

But, previous translation repeatedly explains same code snippets. Here is example: 
> 끝으로, 새 블로그 글을 작성한 다음에 post_detail페이지로 이동할 수 있으면 좋겠죠? 이 작업을 하려면 한 가지를 더 불러와야 해요.
> ```python
> from django.shortcuts import redirect
> ```
> 위 코드를 여러분의 파일 맨 위에 추가하세요. 그리고 새로 작성한 글을 볼 수 있도록 post_detail페이지로 가라고 수정합시다.
> 
> ```python
> # blog/views.py
> from django.shortcuts import redirect
> ```
> post_detail은 우리가 이동해야 할 뷰의 이름이에요 post_detail 뷰 는 pk변수가 필요한 거 기억하고 있겠죠? pk=post.pk를 사용해서 뷰에게 값을 넘겨줄 거에요. 여기서 post는 새로 생성한 블로그 글이에요.